lib/standalone.js

```diff
@@ -170,11 +170,7 @@ module.exports = function(
       });
   }
 
-  let fileList = files;
-
-  if (typeof fileList === "string") {
-    fileList = [fileList];
-  }
+  let fileList = [].concat(files).map(file => file.replace(/\\/g, "/"));
```

Comment:
It can potentially output linux paths on windows, we need test this

Reply:
I did not understand what you exactly mean =(
If you mean that `stylelint` can *try* to output fixed code to wrong (linux-like) file path, than we have such tests, see https://github.com/stylelint/stylelint/blob/master/system-tests/fix/fix.test.js#L60

Reply:
So I found only one workaround for

> There is no way to pass `\` in file name

and

> no ability to use backslashes for escaping characters

allow users escape with double backslashes only 🙀